bugsnag/src/test/java/com/bugsnag/SerializerTest.java

```diff
@@ -0,0 +1,24 @@
+package com.bugsnag;
+
+import static org.junit.Assert.assertEquals;
+
+import com.bugsnag.serialization.SerializationException;
+import com.bugsnag.serialization.Serializer;
+import com.bugsnag.util.CustomSerializer;
+
+import org.junit.Test;
+
+import java.io.ByteArrayOutputStream;
+
+public class SerializerTest {
+
+    @Test
+    public void testCustomSerializer() throws SerializationException {
+        Serializer customSerializer = new CustomSerializer();
```

Comment:
You're not really testing anything here - the class you're calling is a test implementation, so it's just asserting that the compiler can cope with interfaces :) 

Instead, I'd put this test in with the Configuration unit tests and have your `writeToStream` implementation (suggest using an anonymous class for readability of the test) assert that it's called.